packages/shared-components/src/customComponents/components/Fodselsnummer.jsx

```diff
@@ -50,10 +51,17 @@ export default class Fodselsnummer extends TextFieldComponent {
   validateFnr(fnrTekstWithMiddleSpace) {
```

Comment:
For å oppnå bakoverkompatibilitet, ta vare på den gamle funksjonen, gjerne med en kommentar om at denne er der for å sikre bakoverkompatibilitet for eldre skjemaer. Legg ny valideringslogikk i ny funksjon med nytt navn.